src/flow.h

```diff
@@ -301,6 +301,8 @@ typedef struct FlowCnf_
     uint32_t timeout_new;
     uint32_t timeout_est;
 
+    uint32_t elephant_flow_rate;
```

Comment:
It hurts to increase this structure...

Reply:
I see. What is your suggestion?

Reply:
What is elephant flow rate ? Cannot we compute it on demand instead of storing it ?

Reply:
What is elephant flow rate ? Cannot we compute it on demand instead of storing it ?

Reply:
Just checked the code.
Q: Why does it hurt to increase this struct? It does not seem to be referenced/passed around inside other crucial structs and only stores config options wrt flow.

Reply:
Sorry, `FlowCnf_`is ok to expand, it is the other change below